### PR TITLE
Safe filter fields in AQL

### PR DESCRIPTION
**Pull request by MrCreosote**
The classic mistake of thinking since you control the fields you can just use dot notation. Nope, some sample fields have colons in them which borks things up.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # KBase Collections Release Notes
 
+## 0.1.2
+
+* Fixed a bug that caused requests with filters to fail for filter keys containing colons.
+* Adds eggNOG mappper as an experimental tool in the pipeline. It currently does not integrate
```
Comment (Xiangs18) [inline RELEASE_NOTES.md:6]: mapper?

Comment (MrCreosote) [reply RELEASE_NOTES.md]: fixed

Comment (Xiangs18) [reply RELEASE_NOTES.md]: 👍

**Review by Xiangs18 — APPROVED** [review]
LGTM!

**Pull request merged**